ui/src/models/model.ts

```diff
@@ -74,3 +67,20 @@ export interface IFeatureLineage {
   relations: any;
 }
 
+export interface IUserRole {
+  id: number;
+  scope: string;
+  userName: string;
+  roleName: string;
+  createTime: string;
+  createReason: string;
+  deleteTime?: any;
+  deleteReason?: any;
+}
+
+export interface RoleForm {
```

Comment:
Though i am not a big fan of Ixxx. But we should follow the convention. So this prob should be IRoleForm

Reply:
I think we don't need to Ixxx here as there is not scenario in ui code requires different implementations. I will do a clean up to remove Ixxx pattern